recipes/conversation_watsonxassistant/package.json

```diff
@@ -27,7 +27,9 @@
     "eslint-plugin-import": "^2.22.1"
   },
   "dependencies": {
+    "@iarna/toml": "^2.2.5",
     "ibm-watson": "^9.0.0",
-    "tjbot": "^2.0.2"
+    "import-meta-resolve": "^4.1.0",
+    "tjbot": "^2.0.1"
```

Comment:
Should be `^3.0.0`, the recipe won't work with earlier versions of `node-tjbotlib`.